packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -243,6 +246,36 @@ class _FlutterValidator extends DoctorValidator {
   }
 }
 
+class _HostExecutableValidator extends DoctorValidator {
+  _HostExecutableValidator() : super('Host Executable Compatibility');
+
+  bool _genSnapshotRuns(String genSnapshotPath) {
+    final kErrorExitCode = 255;
```

Comment:
Done. I had taken the name directly from the gen_snapshot source code but I think your name is better.